app/Console/Commands/UpdateOrchids.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use App\Profile;
+use App\ProfileData;
+
+
+class UpdateOrchids extends Command
```

Comment:
This should be "UpdateOrcids" (without the "h"). Likewise for the filename and signature on line 17.